packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -125,12 +117,7 @@ class GoogleSignIn {
   /// The [hostedDomain] argument specifies a hosted domain restriction. By
   /// setting this, sign in will be restricted to accounts of the user in the
   /// specified domain. By default, the list of accounts will not be restricted.
-  ///
-  /// The [clientId] argument specifies the ID that was assigned to your app
-  /// when it was registered with Google. It is optional; if unspecified, it
-  /// will be deduced based on your project's metadata (e.g.
-  /// `GoogleServices-Info.plist` on iOS).
-  GoogleSignIn({this.scopes, this.hostedDomain, this.clientId});
+  GoogleSignIn({this.scopes, this.hostedDomain});
```

Comment:
The current released version (`0.0.6`) doesn't have the clientId API in Dart - it was added last Friday and hasn't been released yet.  So we're good :-)